src/confdb/confdb.c

```diff
@@ -1310,9 +1310,15 @@ static int confdb_get_domain_internal(struct confdb_ctx *cdb,
         }
     } else {
         /* default */
-        domain->case_sensitive = true;
-        domain->case_preserve = true;
+        if (strcasecmp(domain->provider, "ad") == 0) {
```

Comment:
OK, I think I was confused because we already set this in `ad_get_common_options`. I guess the tools don't call `ad_get_common_options` though. Correct?